tests/app/models.py

```diff
@@ -17,6 +17,11 @@ class CategoryQueryset(models.QuerySet):
 
 
 class Category(models.Model):
+    """
+    Abstract Category with custom manger required
+    to test patching of parent model managers.
+    """
+
```

Comment:
Can you move this comment to AbstractArticle (and amend it to reflect it's new location)?